app/representers/api/image_representer.rb

```diff
@@ -2,13 +2,16 @@
 
 class Api::ImageRepresenter < Roar::Decorator
   include Roar::JSON
+  include HalApi::Representer::FormatKeys
 
-  property :url
+  property :href
+  property :url, writeable: false # TODO: deprecate in favor of href ... but Castle scrapes this field
   property :original_url, writeable: false
```

Comment:
Note: since I added `HalApi::Representer::FormatKeys` - this property now ends up camelcased `originalUrl`.